healpix_alchemy/point.py

```diff
@@ -26,9 +26,24 @@ def __init__(self, *args, ra=None, dec=None, **kwargs):
     ra = Column(Float)
     dec = Column(Float)
 
+    x = Column(
```

Comment:
Maybe somehow indicate in the doc that these do not increase the DB size?

Reply:
I didn't actually measure the database size. We have the times of unit tests reported; it would be great to also report the size of the database. What's the easiest way to get the database size in SQLAlchemy or PostgreSQL?

Reply:
Hmm looks like calls to postgresql:
https://writeonly.wordpress.com/2010/06/11/snippet-size-of-postgresql-tables-using-sqlalchemy/

Reply:
Also https://wiki.postgresql.org/wiki/Disk_Usage

Reply:
Well, adding indices _does_ increase the table size.